openai/info.json

```diff
@@ -1,113 +1,207 @@
-                    "tooltip": "Specify the ID of the GPT model to use for the chat completion. Currently, only gpt-3.5-turbo and gpt-3.5-turbo-0301 are supported. By default it is set to gpt-3.5-turbo."
-                },
-                {
-                    "title": "Temperature",
-                    "type": "text",
-                    "name": "temperature",
-                    "required": false,
-                    "visible": true,
-                    "editable": true,
-                    "description": "Specify the sampling temperature between 0 and 2. Higher values like 0.8 makes the output more random, while lower values like makes it more focused and deterministic. NOTE: It is recommended to use either this parameter or Top Probability parameter, not both. By default, it is set to 1.",
-                    "tooltip": "Specify the sampling temperature between 0 and 2. Higher values like 0.8 makes the output more random, while lower values like makes it more focused and deterministic. NOTE: It is recommended to use either this parameter or Top Probability parameter, not both. By default, it is set to 1."
-                },
-                {
-                    "title": "Top Probability",
-                    "type": "text",
-                    "name": "top_p",
-                    "required": false,
-                    "visible": true,
-                    "editable": true,
-                    "description": "Specify the top probability, an alternative to sampling with temperature, also called nucleus sampling. The model considers the results of the tokens with top_p probability mass. So 0.1 means only the tokens comprising the top 10% probability mass are considered. NOTE: It is recommended to use either this parameter or Temperature parameter, not both. By default, it is set to 1.",
-                    "tooltip": "Specify the top probability, an alternative to sampling with temperature, also called nucleus sampling. The model considers the results of the tokens with top_p probability mass. So 0.1 means only the tokens comprising the top 10% probability mass are considered. NOTE: It is recommended to use either this parameter or Temperature parameter, not both. By default, it is set to 1."
-                },
+    "operations":[
+       {
+          "title":"Create a chat completion",
+          "operation":"chat_completions",
+          "annotation":"chat_completions",
+          "description":"Generates responses for chat messages using a pre-trained deep learning model.",
+          "parameters":[
+             {
+                "title":"Chat Type",
+                "description":"Select whether it is a single question and answer or a conversation",
+                "tooltip":"Select whether it is a single question and answer or a conversation",
+                "required":true,
```

Comment:
this being a required parameter will cause existing playbooks using the previous connector version to fail when the connector is upgraded.